### DISCUSSION TITLE
Can't get "y" key right

### DISCUSSION THREAD

**Discussion by JoeyWangTW** [aerickt/lapwing-for-beginners#6] [Q&A] [answered]
I'm learning with the book, thanks for the work! Very useful!
In chapter 6, I'm trying to get `y` using `KWH` but my plover doesn't seem to show the same combination.
Here's a screenshot of the lookup, am I missing any setup?
 
<img width="408" alt="Screenshot 2024-06-24 at 8 35 24 AM" src="https://github.com/aerickt/lapwing-for-beginners/assets/11681724/4dee31f2-ae6b-4d3b-a93c-1ea0918c6642">

**Comment by aerickt** [comment, marked as answer]
This was a very recent change actually, sorry about that 😅
See the instructions here to update your dictionary: https://lapwing.aerick.ca/Chapter-05.html#dictionary-updates

**Reply by JoeyWangTW** [reply]
Got it to work, thanks!
